pkg/services/sqlstore/migrator/mysql_dialect.go

```diff
@@ -91,7 +91,11 @@ func (db *MySQLDialect) SQLType(c *Column) string {
 
 	switch c.Type {
 	case DB_Char, DB_Varchar, DB_NVarchar, DB_TinyText, DB_Text, DB_MediumText, DB_LongText:
-		res += " CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci"
+		if c.IsLatin {
```

Comment:
utf8mb4 is 4 bytes for each character, rather than 1 -- usually fine, but with the long "key" for an index, this gets too big 😬